File: Palindrome-Number.java
class Solution {
    public boolean isPalindrome(int x) {
        int rev=0,t=x;
        while(x>0)
        {
            int digi=x%10;
            x=x/10;
            rev=rev*10+digi;
        }
        return(rev==t);
    }
}

File: Palindrome-Number.c
bool isPalindrome(int x) 
{
    if(x<0)
    return false;
    long int rev=0;
    long int temp=x;
    while(x!=0)
    {
        rev = (rev*10) + x%10;
        x=x/10;
    }
    if(temp==rev)
    return true;
    return false;
}